src/dsp_tools/utils/xmlupload/stash/stash_models.py

```diff
@@ -49,3 +46,60 @@ def make(tups: list[tuple[XMLResource, StandoffStashItem]]) -> StandoffStash | N
             else:
                 res_2_stash_items[xmlres.id].append(stash_item)
         return StandoffStash(res_2_stash_items, res_2_xmlres)
+
+
+@dataclass(frozen=True)
+class LinkValueStashItem:
+    """Holds information about a single stashed link value."""
+
+    res_id: str
+    res_type: str
+    prop_name: str
+    target_id: str
+
+
+@dataclass(frozen=True)
+class LinkValueStash:
+    """Holds information about a number of stashed link values, organized by resource instance."""
```

Comment:
I'm not sure if the terminology is clear enough. I often refer to this as `resptr-prop`. Could you perhaps add in the docstring that LinkValue = resptr?

Reply:
I'm reluctant to perpetuate this horrible terminology we have inherited 😉   
No, seriously, of course I can do that, but it is definitely my goal to replace as much of the terminological idiosyncrasies of dsp-tools by the terms we use in the rest of our system. So on the long run, I will try to phase out "resptr"